host/traceh.c

```diff
@@ -201,8 +202,11 @@ static bool _escape_characters(
                         log_msg_escaped[idx] = '\0';
                         return false;
                     }
-                    sprintf(
-                        (char*)&log_msg_escaped[idx], "u%04hhx", log_msg[i]);
+                    sprintf_s(
+                        (char*)&log_msg_escaped[idx],
+                        msg_size,
```

Comment:
I think this should be `msg_size - idx` right? Because that is whats left of the buffer